contracts/treasury/GovernorCountingOverridable.sol

```diff
@@ -0,0 +1,229 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorVotesQuorumFractionUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/interfaces/IERC5805Upgradeable.sol";
+
+import "../bonding/libraries/EarningsPool.sol";
+import "../bonding/libraries/EarningsPoolLIP36.sol";
+
+import "../Manager.sol";
+import "../IController.sol";
+import "../rounds/IRoundsManager.sol";
+import "../bonding/IBondingCheckpoints.sol";
+
+interface IVotes is IERC5805Upgradeable {
```

Comment:
Move this to own file so that BondingCheckpointsVotes doesn't need to ref this file? Was a bit confusing to see a ref to this file in BondingCheckpointsVotes despite BondingCheckpointsVotes not directly depending on GovernorCountingOverridable.

Reply:
Moved on https://github.com/livepeer/protocol/pull/614/commits/d518cd6058e87c7bc10c8ecfc05696e8c30655e8 in the previous PR, when merging `BondingCheckpoints with `...Votes`